### DISCUSSION TITLE
Does not load the network graph

### DISCUSSION THREAD

**Discussion by TheBigEye** [community/community#12286] [Other Feature Feedback, Questions, & Ideas] [answered]
I have noticed that in my repositories the network grap does not load, that is, it loads the date bar and it can be moved, but the content such as commits and branches is not shown

![image](https://user-images.githubusercontent.com/63316583/156340726-fd7cc7d8-e9c5-49f3-9821-bc21eff63b11.png)

**Operating system:** Windows 11 Pro 22000.527
**Browser:** Microsoft edge 98.0.1108.62 64Bits

**Comment by patideepak** [comment]
@github-staff Please look into this. our team is facing this issue since last one week now. There is some major UI issue which is breaking the network graph. This works on few of the screen resolution and throwing error to few of them.

**Comment by akinomyoga** [comment]
This is a duplicate of #12061.

**Comment by TheBigEye** [comment, marked as answer]
Now it works correctly
